README.md

```diff
@@ -1,66 +1,55 @@
-        <img src="https://img.shields.io/github/downloads/MonitorControl/MonitorControl/total.svg" alt="downloads"/>
-    </a>
-    <!-- version -->
-    <a href="https://github.com/MonitorControl/MonitorControl/releases/latest">
-        <img src="https://img.shields.io/github/release/MonitorControl/MonitorControl.svg" alt="latest version"/>
-    </a>
-    <!-- license -->
-    <a href="https://github.com/MonitorControl/MonitorControl/blob/master/License.txt">
-        <img src="https://img.shields.io/github/license/MonitorControl/MonitorControl.svg" alt="license"/>
-    </a>
-    <!-- platform -->
-    <a href="https://github.com/MonitorControl/MonitorControl">
-        <img src="https://img.shields.io/badge/platform-macOS-lightgrey.svg" alt="platform"/>
-    </a>
-</div>
-
+<!-- downloads -->
+<a href="https://github.com/MonitorControl/MonitorControl/releases">
+<img src="https://img.shields.io/github/downloads/MonitorControl/MonitorControl/total.svg?style=flat" alt="downloads"/>
+</a>
+<!-- version -->
+<a href="https://github.com/MonitorControl/MonitorControl/releases">
+<img src="https://img.shields.io/github/release-pre/MonitorControl/MonitorControl.svg?style=flat" alt="latest version"/>
+</a>
+<!-- license -->
+<a href="https://github.com/MonitorControl/MonitorControl/blob/master/License.txt">
+<img src="https://img.shields.io/github/license/MonitorControl/MonitorControl.svg?style=flat" alt="license"/>
+</a>
+<!-- platform -->
+<a href="https://github.com/MonitorControl/MonitorControl">
+<img src="https://img.shields.io/badge/platform-macOS-lightgrey.svg?style=flat" alt="platform"/>
+</a>
+<!-- backers -->
+<a href="https://opencollective.com/monitorcontrol">
+<img src="https://opencollective.com/monitorcontrol/tiers/badge.svg" alt="backers"/>
+</a>
+    
 <br/>
-
-<div align="center">
-    <img src="./.github/menulet.png" width="200" alt="menulet screenshot"/>
-    <br/><br/>
-    <img src="./.github/menugeneral.png" width="299" alt="general screenshot"/><img src="./.github/menukeys.png" width="299" alt="keys screenshot"/>
-    <img src="./.github/menudisplay.png" width="299" alt="display screenshot"/>
-    <img src="./.github/menuadvanced.png" width="299" alt="advanced screenshot"/>
-
 <br/>
 
-_Bonus: Using keyboard keys displays the native osd_
-
-<img src="./.github/osd.jpg" width="500" align="center" alt="osd screenshot"/>
 </div>
 
-## Download
+<img src=".github/screenshot.png" width="840" alt="Screenshot"/><br/>
 
-Go to [Release](https://github.com/MonitorControl/MonitorControl/releases/latest) and download the latest `.dmg`
+<hr>
 
-## Installing with Homebrew Cask
+## Download
 
-You can also install MonitorControl with [Homebrew Cask](https://github.com/Homebrew/homebrew-cask).
+Go to [Releases](https://github.com/MonitorControl/MonitorControl/releases) and download the latest `.dmg`
 
-```bash
-$ brew install --cask monitorcontrol
-```
+## Compatibility
+
+- macOS Mojave (`10.14`) and up.
```

Comment:
Perhaps a note for users with older versions on which version they can use?

Reply:
Yes, I agree.

Reply:
I added this to the readme:

- macOS Mojave (`10.14`) and up _(note: you can download [version 2.1.0](https://github.com/MonitorControl/MonitorControl/releases/tag/v2.1.0) for macOS Sierra `10.12` support)_